### PR TITLE
Hide velocity section

### PR DESCRIPTION
**Pull request by kntsoriano**
Screen Shot 2021-08-31 at 2 20 29 PM

### CODE REVIEW

**Comment by iHiD** [comment]
Just working this one out with @taiyab. We might add something else to the RHS so it doesn't look so bland.

**Comment by iHiD** [comment]
@kntsoriano TODOs in here for you now :)

File: app/javascript/components/journey/overview/learning-section/TrackSummary.tsx [hunk 1]
```diff
@@ -32,16 +34,64 @@ export const TrackSummary = ({
...
-            in the last 6 months
-          </div>
+        <div className="date-area">
+          <GraphicalIcon icon="entry" />
+          <h4 className="journey-h3">
+            {timeFormat(track.startedAt, 'DD MMM YYYY')}
+          </h4>
+          <p className="text-h6">When you joined the {track.title} Track</p>
+          <p>
+            You started working through the {track.title} Track{' '}
```
Comment (kntsoriano) [inline app/javascript/components/journey/overview/learning-section/TrackSummary.tsx:44]: @iHiD I changed the copy a bit on this in order for us to use our `fromNow` helper.

**Comment by iHiD** [comment]
Good work, team!

**Pull request merged**